lib/Conversion/TorchToLinalg/Pooling.cpp

```diff
@@ -1616,6 +1618,631 @@ class ConvertAtenAdaptivePoolOp : public OpConversionPattern<OpTy> {
 };
 } // namespace
 
+namespace {
+template <typename OpTy, typename PoolingOpTy, int Dim>
+class ConvertRoiAlignOp : public OpConversionPattern<OpTy> {
+public:
```

Comment:
nit: For patterns that don't have any state, we'd typically use:
```suggestion
struct ConvertRoiAlignOp : final OpConversionPattern<OpTy> {
```